Platform/ElkhartlakeBoardPkg/Script/StitchIfwiConfig.py

```diff
@@ -101,6 +102,7 @@ def check_parameter(para_list):
         'dnx'       -- Enable DNX capsule binary build for IFWI updating via DNX
         'pt'        -- Enable Proof Test configuration
         'isi_conf'  -- Enable ISI FW configuration file
+        'debug'     -- Enable DAM and DCI configuration

```

Comment:
I think we should state this should only be used for debug and should not be enabled in the final production.

Reply:
Closing this PR due to git issue...submitted patch to another branch below:
#1215